python/lsst/ts/salobj/configurable_csc.py

```diff
@@ -428,7 +431,8 @@ def get_config_pkg():
         """
         raise NotImplementedError()
 
-    def get_default_config_dir(self):
+    # was public - def get_default_config_dir(self):
+    def __get_default_config_dir(self):
```

Comment:
Please just use one leading underscore. I prefer not to invoke the Python name munging mechanism unless it is truly necessary. I would rather rely on users to be sensible, and allow overriding a method if a user has a good reason to do so.